stock_request_picking_type/tests/test_stock_request.py

```diff
@@ -7,46 +7,11 @@
 
 
 class TestStockRequest(common.TransactionCase):
-    def setUp(self):
-        super(TestStockRequest, self).setUp()
-
-        # common models
-        self.stock_request = self.env["stock.request"]
-        self.request_order = self.env["stock.request.order"]
-
-        # refs
-        self.stock_request_user_group = self.env.ref(
-            "stock_request.group_stock_request_user"
-        )
-        self.main_company = self.env.ref("base.main_company")
-        self.warehouse = self.env.ref("stock.warehouse0")
-        self.default_picking_type = self.env.ref(
-            "stock_request_picking_type.stock_request_order"
-        )
-
-        # common data
-        self.company_2 = self.env["res.company"].create(
-            {"name": "Comp2", "parent_id": self.main_company.id}
-        )
-        self.company_2_address = (
-            self.env["res.partner"]
-            .with_context(company_id=self.company_2.id)
-            .create({"name": "Peñiscola"})
-        )
-        self.product = self._create_product("SH", "Shoes", False)
-        self.product_company_2 = self._create_product(
-            "SH_2", "Shoes", self.company_2.id
-        )
-        self.stock_request_user = self._create_user(
-            "stock_request_user",
-            [self.stock_request_user_group.id],
-            [self.main_company.id, self.company_2.id],
-        )
-
-    def _create_user(self, name, group_ids, company_ids):
+    @classmethod
+    def _create_user(cls, name, group_ids, company_ids):
         return (
-            self.env["res.users"]
-            .with_context({"no_reset_password": True})
+            cls.env["res.users"]
+            .with_context(**{"no_reset_password": True})
```

Comment:
You can avoid such writing by using arguments directly:
`.with_context(no_reset_password=True)`

Reply:
Fix it!